scripts/test/lld.py

```diff
@@ -33,11 +33,15 @@ def test_wasm_emscripten_finalize():
 
   for wast_path in files_with_pattern(options.binaryen_test, 'lld', '*.wast'):
     print '..', wast_path
+    is_passive = '.passive.' in wast_path
     mem_file = wast_path + '.mem'
     extension_arg_map = {
-      '.out': [],
-      '.mem.out': ['--separate-data-segments', mem_file],
+        '.out': [],
     }
+    if not is_passive:
+      extension_arg_map.update({
+          '.mem.out': ['--separate-data-segments', mem_file],
```

Comment:
What's the indentation we are supposed to use here? This file looks like it's 2 space indented, but this is 4 spaces indented.

Reply:
I think it uses four here because it is a continuation of an expression from the previous line. Flake8 doesn't complain at any rate.